apps/src/templates/certificates/Certificate.jsx

```diff
@@ -153,28 +203,46 @@ function Certificate(props) {
           {i18n.congratsCertificateHeading()}
         </Heading1>
       </div>
-      {tutorial && (
+      {courseName && (
         <LargeChevronLink
-          link={`/s/${tutorial}`}
+          link={`/s/${courseName}`}
           linkText={i18n.backToActivity()}
         />
       )}
       <div className={style.certificateContainer}>
-        <div id="uitest-certificate" className={certificateStyle}>
-          <BackToFrontConfetti
-            active={personalized}
-            className={style.confetti}
-          />
-          <a href={certificateShareLink}>
-            {
-              // TODO: A11y279 (https://codedotorg.atlassian.net/browse/A11Y-279)
-              // Verify or update this alt-text as necessary
-            }
-            <img src={imgSrc} alt="" />
-          </a>
+        <div
+          id="uitest-certificate"
+          style={{
+            width: '50%',
+            display: 'flex',
+          }}
+        >
+          {
+            <BackToFrontConfetti
+              active={personalized}
+              className={style.confetti}
+            />
+          }
+          <swiper-container ref={swiperRef} class={style.swiperContainer}>
+            {certificateData.map(image => (
+              <swiper-slide key={image.courseName} class={style.swiperSlide}>
+                <a href={getCertificateSharePath(image.courseName)}>
+                  {
+                    // TODO: A11y279 (https://codedotorg.atlassian.net/browse/A11Y-279)
+                    // Verify or update this alt-text as necessary
+                  }
+                  <img
+                    src={getCertificateImagePath(image.courseName)}
+                    alt=""
```

Comment:
Created https://codedotorg.atlassian.net/browse/ACQ-1554 to add alt-text here. I decided to make it a separate task so that I can unify a few different places that need alt text.